### PR TITLE
Initial ActivityPub Maint WG charter template.

### PR DESCRIPTION
**Pull request by dmitrizagidulin**
This is a sample (to be discussed with the CG) charter template for the continuing discussion of a potential Working Group, based on a minimal stock W3C template.

Notes:
* Narrowly scoped as an ActivityPub (and ActivityStreams and ActivityVocabulary on which it depends) Maintenance group.

### CODE REVIEW

File: ap-maintenance-wg-charter.html [hunk 1]
```diff
@@ -0,0 +1,529 @@
...
+      </section>
+
+	<section id="success-criteria">
+	  <h2>Success Criteria</h2>
+
+    <p>
+      In order to update the Recommendation, each substantive change is expected
+      to have <a href="https://www.w3.org/Consortium/Process/#implementation-experience">at least
+      two independent implementations</a> of every feature defined in the
+      specification, where interoperability can be verified by passing open test suites.
```
Comment (csarven) [inline ap-maintenance-wg-charter.html:243]:
```suggestion
      specification, where interoperability can be verified by passing open test suites, and two or more implementations interoperating with each other.
```

**Comment by bumblefudge** [comment]
oh, another example just occurred to me: if the candidate analysis from the e2ee CG work item lands on profiling an IETF protocol, wouldn't _that_ WG be the one to continue work on after CG report? the best WG to do further work at might not always be at w3c!

File: ap-maintenance-wg-charter.html (hunk 1 above)
Comment (dmitrizagidulin) [reply ap-maintenance-wg-charter.html:243]: +1, merging this suggestion. (two+ implementations is a common requirement for w3c groups and beyond)

**Comment by dmitrizagidulin** [comment]
Multiple weeks review period passed; discussed it during WG call, also sent out notifications on mailing list, forum and fediverse. No objections from the community.
Since this is just an initial draft proposal, merging (to continue the conversation).

**Pull request merged**